101-perforce-helix-core-server/azuredeploy.json

```diff
@@ -0,0 +1,357 @@
+    "subnet1Name": "Subnet-1",
+    "subnet1Prefix": "10.0.0.0/24",
+    "numberOfInstances": 1,
+    "linuxscripturi": "[uri(parameters('_artifactsLocation'), concat('scripts/configure-linux.sh', parameters('_artifactsLocationSasToken')))]",
+    "virtualMachineSize": "[parameters('VMSize')]",
+    "imageReference": {
+      "Ubuntu 18.04 LTS": {
+        "publisher": "Canonical",
+        "offer": "UbuntuServer",
+        "sku": "18.04-LTS",
+        "version": "latest"
+      },
+      "CentOS 7.5": {
+        "publisher": "OpenLogic",
+        "offer": "CentOS",
+        "sku": "7.5",
+        "version": "latest"
+      },
+      "RHEL 7.x": {
+        "publisher": "RedHat",
+        "offer": "RHEL",
+        "sku": "7-LVM",
+        "version": "latest"
+      }
+    },
+    "dataDiskSize": "[parameters('dataDiskSize')]",
+    "publicIpName": "[substring(concat(uniqueString(resourceGroup().id, deployment().name)), 0, 6)]",
+    "LinuxScriptParameters": "[concat(' -w ''', parameters('helix_admin_password'), ''' -p ''', parameters('p4Port'), ''' -s ''', parameters('swarmPort'), '''')]",
+    "LiCmdWrapper": "[concat('bash ./configure-linux.sh ', variables('LinuxScriptParameters'))]",
+    "LinuxsecurityRules": [
+      {
+        "name": "ssh-rule",
+        "properties": {
+          "description": "Allow SSH",
+          "protocol": "Tcp",
+          "sourcePortRange": "*",
+          "destinationPortRange": "22",
+          "sourceAddressPrefix": "[parameters('source_CIDR')]",
+          "destinationAddressPrefix": "*",
+          "access": "Allow",
+          "priority": 100,
+          "direction": "Inbound"
+        }
+      },
+      {
+        "name": "web-rule",
+        "properties": {
+          "description": "Allow WEB",
+          "protocol": "Tcp",
+          "sourcePortRange": "*",
+          "destinationPortRange": "[parameters('swarmPort')]",
+          "sourceAddressPrefix": "Internet",
+          "destinationAddressPrefix": "*",
+          "access": "Allow",
+          "priority": 102,
+          "direction": "Inbound"
+        }
+      },
+      {
+        "name": "p4d-rule",
+        "properties": {
+          "description": "Allow WEB",
+          "protocol": "Tcp",
+          "sourcePortRange": "*",
+          "destinationPortRange": "[parameters('p4Port')]",
+          "sourceAddressPrefix": "Internet",
+          "destinationAddressPrefix": "*",
+          "access": "Allow",
+          "priority": 103,
+          "direction": "Inbound"
+        }
+      }],
+    "linuxConfiguration": {
+      "disablePasswordAuthentication": true,
+      "ssh": {
+        "publicKeys": [
+          {
+            "path": "[concat('/home/', parameters('adminUsername'), '/.ssh/authorized_keys')]",
+            "keyData": "[parameters('adminSSHPubKey')]"
+          }
+        ]
+      }
+    }
+  },
+  "resources": [
+    {
+      "apiVersion": "2019-05-01",
+      "name": "pid-6c922c23-cc65-4b76-8290-74feb0f3c476",
```

Comment:
is this registered?

Reply:
I just copied that field. Have changed it to a simple string name: "helix_core_server". That OK?